src/main/java/com/refinedmods/refinedstorage/item/blockitem/ColoredBlockItem.java

```diff
@@ -0,0 +1,35 @@
+package com.refinedmods.refinedstorage.item.blockitem;
+
+import com.refinedmods.refinedstorage.block.BaseBlock;
+import net.minecraft.block.Block;
+import net.minecraft.item.DyeColor;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.text.IFormattableTextComponent;
+import net.minecraft.util.text.ITextComponent;
+import net.minecraft.util.text.StringTextComponent;
+import net.minecraft.util.text.TranslationTextComponent;
+import net.minecraftforge.fml.RegistryObject;
+
+
+public class ColoredBlockItem extends BaseBlockItem {
+    private final DyeColor color;
+    private final RegistryObject<? extends Block> blockForTranslation;
+
+    public ColoredBlockItem(BaseBlock block, Properties builder, DyeColor color, RegistryObject<? extends Block> blockForTranslation) {
```

Comment:
Maybe just pass the the ITextComponent here instead of the whole `blockForTranslation`.

Reply:
This is not resolved